### PR TITLE
Update docs: aws-lambda plugin according to PR#2823

### PR DESCRIPTION
**Pull request by akh00**

### CODE REVIEW

File: app/plugins/aws-lambda.md [hunk 1]
```diff
@@ -61,6 +61,10 @@ form parameter                             | default | description
 `config.log_type`<br>*optional*            | `Tail`  | The [`LogType`](http://docs.aws.amazon.com/lambda/latest/dg/API_Invoke.html#API_Invoke_RequestSyntax) to use when invoking the function. By default `None` and `Tail` are supported
 `config.timeout`<br>*optional*             | `60000` | An optional timeout in milliseconds when invoking the function
 `config.keepalive`<br>*optional*           | `60000` | An optional value in milliseconds that defines how long an idle connection will live before being closed
+`config.forward_request_body`<br>*optional*           | `false` | An optional value that defines whether the "raw" body is to be sent in the `request_body` field of the json encoded request. If the body arguments can be parsed they are still sent in the separate `request_body_args` field of the request.
```
Comment (decoursin) [inline app/plugins/aws-lambda.md]: Better: ```... True or false, if the body arguments can be parsed they are still sent in the `request_body_args` field of the request.```

File: app/plugins/aws-lambda.md [hunk 2]
```diff
@@ -61,6 +61,10 @@ form parameter                             | default | description
...
+`config.forward_request_headers`<br>*optional*           | `false` | An optional value that defines whether the origial HTTP request headers are to be sent as a map in the `request_headers` field of the json encoded request.
+`config.forward_request_method`<br>*optional*           | `false` | An optional value that defines whether the origial HTTP request method name (`POST`, `GET` etc.) is to be sent in the `request_method` field of the json encoded request.
```
Comment (decoursin) [inline app/plugins/aws-lambda.md]: Should this default to `true`?

File: app/plugins/aws-lambda.md [hunk 3]
```diff
@@ -61,6 +61,10 @@ form parameter                             | default | description
 `config.log_type`<br>*optional*            | `Tail`  | The [`LogType`](http://docs.aws.amazon.com/lambda/latest/dg/API_Invoke.html#API_Invoke_RequestSyntax) to use when invoking the function. By default `None` and `Tail` are supported
 `config.timeout`<br>*optional*             | `60000` | An optional timeout in milliseconds when invoking the function
 `config.keepalive`<br>*optional*           | `60000` | An optional value in milliseconds that defines how long an idle connection will live before being closed
+`config.forward_request_body`<br>*optional*           | `false` | An optional value that defines whether the "raw" body is to be sent in the `request_body` field of the json encoded request. If the body arguments can be parsed they are still sent in the separate `request_body_args` field of the request.
+`config.forward_request_headers`<br>*optional*           | `false` | An optional value that defines whether the origial HTTP request headers are to be sent as a map in the `request_headers` field of the json encoded request.
```
Comment (decoursin) [inline app/plugins/aws-lambda.md]: original, not origial.

**Comment by decoursin** [comment]
@akh00 Thank you. I'm :+1:.

**Comment by thibaultcha** [comment]
Thanks! Merged to #508 which is the `release/master` branch and holds documentation updates for the changes currently accepted in Kong's master but not released.

**Comment by decoursin** [comment]
Haha poor @akh00 everything he does gets ignored

**Pull request closed without merging**